addons/l10n_tw_edi_ecpay_pos/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Taiwan - E-invoicing Pos",
+    "countries": ["tw"],
+    "version": "1.0",
+    "category": "Point of sale/Localizations/EDI",
+    "summary": """ECpay E-invoice bridge module for POS""",
+    "description": """
+        This bridge module allows the user to input Ecpay information in pos for sending their invoices to the Ecpay system
+    """,
+    "website": "https://www.odoo.com",
+    "license": "LGPL-3",
+    "depends": [
+        "point_of_sale",
+        "l10n_tw_edi_ecpay",
+    ],
```

Comment:
I think this one and the e-commerce one could be auto-installed if both dependencies are installed.